### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by KseniiaRiabova**
- Added technologies
- Updated project description ( About Section )

### CODE REVIEW

File: README.md
Comment (r1g023) [inline README.md:1]: Nice job @KseniiaRiabova

**Review by r1g023 — COMMENTED** [review]
Nice job @KseniiaRiabova

**Review by r1g023 — APPROVED** [review]
Nice job

**Review by corinnabolon — APPROVED** [review]
Looks great!

File: README.md
Comment (KseniiaRiabova) [reply README.md:1]: Thank you!

**Pull request merged**